packages/rmf-dashboard-framework/docs/getting-started.md

```diff
@@ -0,0 +1,115 @@
+## Getting Started
+
+Welcome to RMF Dashboard Framework! This tutorial will guide you through the initial setup and provide a basic example to help you start building your dashboard application.
+
+### Setup
+
+Prerequisites:
+
+- pnpm
+- Latest Nodejs LTS
+
+Follow https://pnpm.io/installation to install pnpm, you can then install nodejs using pnpm
+
+```bash
+pnpm env use --global lts
+```
+
+Clone this repo and install deps
+
+<!-- TODO(koonpeng): install from npmjs after package is published. -->
+
+```bash
+pnpm create vite -t react-ts basic-dashboard
+cd basic-dashboard
+git clone --depth 1 https://github.com/open-rmf/rmf-web
+```
+
+<!-- TODO(koonpeng): not needed after package is published -->
+
+Modify the following to resolve `rmf-dashboard-framework` as if it is installed from npmjs.
+
+package.json
```

Comment:
```suggestion
`~/basic-dashboard/package.json`
```

Reply:
I'll leave it to you to decide whether an inline code block before a big code block looks OK :sparkles: , or just plain text is enough